### ISSUE TITLE
Play repetition=1 broken for timeScale<0

### ISSUE THREAD

**Issue by Jayemby** [google/model-viewer#3223]
I've just (23/02/22) done a local build to get the `timeScale` property to work (brilliant work, by the way, @futahei ).
However, the `repetitions` option set to 1 in `play()` is broken for negative `timeScale`s

to recreate:
optionally, add a listener / bind a debug function to the `finished` event.
then:
```
modelViewer.timescale = -1;
modelViewer.play({repetitions:1});
```
observe that no animation plays (and that `finished` is immediately triggered). I even tried -1 repetitions but that didn't work either.
to be clear: `modelViewer.play({repetitions:x});` still works like a charm for every positive `x` other than 1 for a negative `timeScale`. In fact, a workaround for this is to use `modelViewer.play({repetitions:0.9});` as it actually gives the desired affect of `modelViewer.play({repetitions:1});`.

Live Demo at: https://garnet-level-cowbell.glitch.me/

Chrome 98.0.4758.102
Windows 11 Home 21H2

**Comment by vaggarath** [comment]
Have you read this ?
https://github.com/google/model-viewer/discussions/3154

**Comment by Jayemby** [comment]
I have indeed. #3154 seems to focus on the timeScale property simply _"not working"_ altogether, due to the feature (commit [8945bb0](https://github.com/google/model-viewer/commit/8945bb0af4874357297ff4b0506e7c06618527e4) 19 Jan #3129) not being yet included in the public build (v1.10.1 14 Jan).

As you can see in my post, I **did** do a local build (as suggested in #3154) to use the feature, and it **is** working as expected in most cases. If you view the live demo (glitch.me) link, you will be able to see that build (and the timeScale feature) present and working well (the demo uses my 23 Feb build and not the v1.10.1 14 Jan release).

My issue is not in getting the overall timeScale feature to work, but that the feature has a bug when `repetitions` is set to 1 in `play()`.
